PRINCIPLES.md

```diff
@@ -241,34 +271,47 @@ Everyone on-call should only acknowledge incidents that they have the ability an
 - Code of Ethics and Professional Conduct from W3C [w3.org/Consortium/cepc](https://www.w3.org/Consortium/cepc/)
 - Color Checker for Accessibility [whocanuse.com](https://www.whocanuse.com)
 
+### Holacracy Ressources
+
+- Holacracy Constitution [github.com/holacracyone/Holacracy-Constitution](https://github.com/holacracyone/Holacracy-Constitution/blob/master/Holacracy-Constitution.md)
+
 ### Technical Guides & Tools
 
-- GPG for Mac [gpgtools.org](https://gpgtools.org)
-- Setting up GPG and Commit Signatures for GitHub [docs.github.com/en/authentication](https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key)
-- 1Password for SSH & Git (incl. signing) [developer.1password.com/docs/ssh](https://developer.1password.com/docs/ssh)
 - npm Best Practices Guide [github.com/ossf/package-manager-best-practices](https://github.com/ossf/package-manager-best-practices/blob/main/published/npm.md)
 - NodeJS Release Schedule [github.com/nodejs/release](https://github.com/nodejs/release#release-schedule)
+- Roadmap learning examples for DevOps [roadmap.sh/devops](https://roadmap.sh/devops) & backend [roadmap.sh/backend](https://roadmap.sh/backend)
 - SPDX License List [spdx.org/licenses](https://spdx.org/licenses/)
 - Super Linter [github/super-linter](https://github.com/github/super-linter)
 - Awesome Broadcasting by the EBU [github.com/ebu/awesome-broadcasting](https://github.com/ebu/awesome-broadcasting)
 - Coordinated Vulnerability Disclosure (CVD) by BSI in Germany [DE] [bsi.bund.de/...](https://www.bsi.bund.de/DE/IT-Sicherheitsvorfall/IT-Schwachstellen/it-schwachstellen_node.html)
 - Use IAM securely [cloud.google.com/iam/docs/using-iam-securely](https://cloud.google.com/iam/docs/using-iam-securely)
 
+#### Security & GPG Tools
+
+- GPG for Mac [gpgtools.org](https://gpgtools.org)
+- Setting up GPG and Commit Signatures for GitHub [docs.github.com/en/authentication](https://docs.github.com/en/authentication/managing-commit-signature-verification/telling-git-about-your-signing-key)
+- 1Password for SSH & Git (incl. signing) [developer.1password.com/docs/ssh](https://developer.1password.com/docs/ssh)
```

Comment:
I would add that one can sign git commits via an _SSH_ key, which reduces the overhead of using GPG for Mac and managing GPG keys when the SSH keys are already available. https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits (so far supported in all major tools such as GitHub, GitLab, GitTea)

Reply:
good point! I clarified it in the text above and added the link